src/styles/main.scss

```diff
@@ -0,0 +1,120 @@
+$watch-size: 400px;
+$pin-size: $watch-size/30;
+
+@mixin watch-hand($w, $h, $clr, $br-clr, $spin-dur) {
+  position: absolute;
+  bottom: 50%;
+
+  box-sizing: border-box;
```

Comment:
Always set `box-sizing` in `* {...}` selector

Reply:
Misha thinks other way, already discussed this with Illia & I don't need it for all elements